.env.example

```diff
@@ -13,3 +13,5 @@ DATA_INGESTION_API_KEY=
 
 CPU_TYPE=ONE_VCPU # ONE_VCPU, TWO_VCPU, FOUR_VCPU, QUARTER_VCPU, HALF_VCPU
 MEMORY_TYPE=TWO_GB # TWO_GB, FOUR_GB, EIGHT_GB, TWELVE_GB, ONE_GB, HALF_GB, SIX_GB
+
+VPC_ID=YOUR_VPC_ID
```

Comment:
**Suggestion:** The example environment variable `VPC_ID` should include a comment explaining what it represents and how to obtain it.
> 📑  **Medium Priority** 
 > [best practice, importance: 5]
```suggestion
VPC_ID=YOUR_VPC_ID # The ID of the VPC to use for deployment. Obtain this from the AWS VPC Creator repository.
```